jenkins/bootstrap.py

```diff
@@ -140,7 +160,9 @@ def checkout(repo, branch, pull):
                 raise
             logging.warning('git fetch failed')
             random_sleep(attempt)
-    call([git, 'checkout', 'FETCH_HEAD'])
+    call([git, 'checkout', '-B', 'test', checkouts[0]])
```

Comment:
checkout[0] will be master or release-1.4 or something right? My original question was how do we ensure this is up to date master (as opposed to whatever the state of master was when pr/123 forked). A secondary question is how do we know that we're using release-1.4 instead of release-1.5 or master?